charts/trusted-artifact-signer/templates/segment-backup-cronjob.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.configs.segment_backup_job.enabled }}
```

Comment:
Do you think it would be better to lock the job / cronjob to the `sigstore-monitoring` namespace and go with the path you suggested or just let uses choose the namespace for everyone of those resources with `configs.segment_backup_job.namespace`. For the 2nd suggestion a potential issue a user could try to pass a namespace that doesnt exist and cause it to fail.